app/templates/auth/create-user.html

```diff
@@ -0,0 +1,84 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Create account – Digital Marketplace{% endblock %}
+
+{% block main_content %}
+
+{% if not token %}
```

Comment:
Again, a straight take from the supplier app and I wondered exactly the same thing.
I thought there might be something I was missing and left it there as a belt-and-braces type approach to error handling (if it never gets called then what's the problem), but now you've said the same thing it makes me more confident in ditching it - I have a feeling it must be a left-over from before the error handling got moved out to `create-buyer-user-error.html`